girder/girder_large_image/web_client/vue/components/CompositeChannels.vue

```diff
@@ -0,0 +1,184 @@
+<script>
+import { Chrome } from 'vue-color';
+
+const CHANNEL_COLORS = {
```

Comment:
For channel colors, we want to have a dictionary of known channel names with predefined colors (see https://github.com/Kitware/UPennContrast/blob/master/src/store/model.ts#L539).  Eventually this could be overridden by the folder-level config (via the .large_image_config.yaml mechanism, but that can be a future PR).  If the channel name isn't present in our map, we would fall back to a list that is much more ordered on primary color channels (https://github.com/Kitware/UPennContrast/blob/master/src/store/model.ts#L474 is an simple way to have such as list, but maybe it could be generated programmatically).  The idea is that the channels would use as independent as possible colors.

Reply:
Changed in [560ec8e](https://github.com/girder/large_image/pull/1086/commits/560ec8e65cfdca9193753d0fdf427867352c3eb7)